server/services/serviceCatalogueService.ts

```diff
@@ -145,7 +142,7 @@ export default class ServiceCatalogueService {
     serviceAreaId?: number
     serviceAreaSlug?: string
     withProducts?: boolean
-  }): Promise<ServiceArea> {
+  }): Promise<StrapiServiceArea> {
```

Comment:
This version of the strapi api has entirely different structure of responses depending on whether its a singular or multiple or embedded response. 

The next step would be to move this over to the same `ServiceArea` representation so we can have consistency and return the same type.

Dealing of which bits of a subtree are populated is a bit of a different kettle of fish - getting type safety over that might be challenging and not worth the complexity.